src/web/crate_details.rs

```diff
@@ -126,7 +126,7 @@ impl CrateDetails {
         .unwrap())
     }
 
-    async fn new(
+    pub(crate) async fn new(
```

Comment:
we don't need this change any more, right?

Reply:
Correct.